src/main/java/net/dries007/tfc/util/Helpers.java

```diff
@@ -473,4 +478,25 @@ public static ResourceLocation readResourceLocation(ByteBuf buf)
     {
         return null;
     }
+
+    public static final DataSerializer<Long> LONG_DATA_SERIALIZER = new DataSerializer<Long>() {
+        public void write(PacketBuffer buf, Long value) {
+            buf.writeLong(value);
+        }
+
+        public Long read(PacketBuffer buf) throws IOException {
+            return buf.readLong();
+        }
+
+        public DataParameter<Long> createKey(int id) {
+            return new DataParameter(id, this);
+        }
+
+        public Long copyValue(Long value) {
+            return value;
+        }
+    };
+    static {
+        DataSerializers.registerSerializer(LONG_DATA_SERIALIZER);
+    }
```

Comment:
Ok. Btw, is it a good place for this stuff? Didn't find where else to put it
I added second static{} because it works only on previous declarations, and I didn't want to put new stuff at the start. I believe they are combined at compile time anyway

Reply:
Done.

Reply:
It's fine to be at `Helpers`, no need to create a class for this.